contracts/Everlong.sol

```diff
@@ -340,12 +372,11 @@ contract Everlong is IEverlong {
         while (!_portfolio.isEmpty()) {
             position = _portfolio.head();
             if (!IHyperdrive(hyperdrive).isMature(position)) {
-                return output;
+                break;
```

Comment:
Suggestion: IMO, this change makes the code harder to reason about and more prone to errors if it were to be changed. When we use `return output;`, it's obvious that execution stops here. With `break`, I have to reason about what happens afterwards.